lang/c.md

```diff
@@ -14,11 +14,23 @@ int main()
 }
 ```
 
-## What is C?
+## ❓ What is C?
 
 C programming is a general-purpose, procedural, imperative computer programming language developed in 1972 by Dennis M. Ritchie at the Bell Telephone Laboratories to develop the UNIX operating system. C programming is considered as the base for other programming languages, that is why it is known as mother language. C language is a system programming language because it can be used to do low-level programming (for example driver and kernel). It is generally used to create hardware devices, OS, drivers, kernels, etc. For example, Linux kernel is written in C. It supports the feature of both low-level and high-level languages.
+<br><br>
 
-## Why one should use C?
+## 🖱️  Application of C
+
+-   Operating Systems &nbsp; &nbsp;(UNIX, Linux, Microsoft Windows)
+-   Embedded Systems &nbsp; &nbsp;(Firmwares, Drivers)
+-   GUI &nbsp; &nbsp;(Adobe Photoshop, Adobe Premiere, Illustrator)
+-   New Programming Platforms &nbsp; &nbsp;(C++, MATLAB, Mathematica)
+-   Compiler Design &nbsp; &nbsp;(Dev-C, Clang C, MINGW, and Apple C)
+-   DataBase Server &nbsp; &nbsp;(MySQL)
+-   Gaming and Animation
```

Comment:
I think the indentation here is wrong. It should be only 1 space.

Reply:
@Panquesito7 actually I saw other indentation of bullet marks in this script are similar like that. I think it is done for more clarification in the raw code

Reply:
@Panquesito7 I did it to make it a bit eye catchy by giving extra space between every point and example